src/pages/docs/getting-started/laravel.mdx

```diff
@@ -0,0 +1,398 @@
+
+<Code>
+```shell
+php artisan make:event PublicMessageEvent
+```
+</Code>
+
+This command will create a `PublicMessageEvent.php` file in the `app/Events` directory. Replace the content of the files with following to handle the broadcasting of messages to a public channel:
+
+<Code>
+```php
+<?php
+
+namespace App\Events;
+
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcastNow;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+/**
+ * Event for broadcasting public messages.
+ */
+class PublicMessageEvent implements ShouldBroadcastNow
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    public $channelName;
+    public $message;
+
+    /**
+     * Create a new event instance.
+     */
+    public function __construct($channelName, $message)
+    {
+        $this->channelName = $channelName;
+        $this->message = $message;
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     *
+     * @return array<\Illuminate\Broadcasting\Channel>
+     */
+    public function broadcastOn(): array
+    {
+        return [new Channel($this->channelName)];
+    }
+}
+```
+</Code>
+
+## Step 3: Add an API route to broadcast <a id="step-3"/>
+
+To handle API requests from frontend, open the `routes/api.php` file and add following route:
+
+<Code>
+```php
+<?php
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use App\Events\PublicMessageEvent;
+
+Route::post('/public-event', function (Request $request) {
+    $channelName = $request->input('channelName');
+    $message = $request->input('message');
+    broadcast(new PublicMessageEvent($channelName, $message));
+    return response()->json(['success' => true, 'message' => 'Event broadcasted']);
+})->middleware('throttle:60,1'); // 60 requests/minute are allowed.
+```
+</Code>
+
+This endpoint `public-event` will accept POST requests with `channelName` and `message` parameters, and broadcast the event to the specified channel.
+
+## Step 4: Test the API route <a id="step-4"/>
+
+To test the API route, first you need a subscriber to a channel. You can use the Ably CLI to subscribe to a channel:
+
+<Code>
+```shell
+ably channels subscribe public:my-first-channel
+```
+</Code>
+
+Now you can publish a message to the channel using the API route you created in Step 3. You can use `curl` or any HTTP client to send a POST request:
+
+<Code>
+```shell
+curl --location --request POST 'localhost:8000/api/public-event' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "channelName":"my-first-channel",
+    "message":"A message sent from my first client!"
+}'
+```
+</Code>
+
+## Step 5: Frontend setup <a id="step-5"/>
+
+Install `Ably`, Ably's fork of `laravel-echo`, and `axios` in your Laravel project:
```

Comment:
Even though it's the import name, I think it's odd to say "Install Ably". We should clarify that it's Pub/Sub